abipy/abio/inputs.py

```diff
@@ -1577,7 +1577,11 @@ def new_with_structure(self, new_structure, scdims=None, verbose=1) -> AbinitInp
             # Rescale nband and k-point sampling
             iscale = int(np.ceil(len(new.structure) / len(self.structure)))
             if "nband" in new:
-                new["nband"] = int(self["nband"] * iscale)
+                # take care of nband in format "*xxx"
```

Comment:
Can you please add a unit test in `abio/tests/test_inputs.py` to make sure that new_with_structure supports `nband="*xxx"?